python-checks/src/test/java/org/sonar/python/checks/UnusedLocalVariableCheckTest.java

```diff
@@ -87,4 +87,40 @@ void exceptClauseQuickFixTest() {
     PythonQuickFixVerifier.verify(check, before, after);
     PythonQuickFixVerifier.verifyQuickFixMessages(check, before, "Remove the unused local variable");
   }
+
```

Comment:
Related to the previous comment, I think having a test where `_` is defined at the top level (thus we don't offer a quick fix) and one where `_` is defined at a class level (where we do offer a quick fix, assuming this indeed doesn't change the behavior of the program) would be nice.